orchestrator/taste-orchestrator.py

```diff
@@ -2471,8 +2545,14 @@ def CreateIncludePaths(
     for baseDir in cyclicSubsystems:
         if os.path.isdir(baseDir + os.sep + baseDir):
             cyclicIncludes += ' -I "../../' + baseDir + os.sep + '"'
+
+    rosBridgeSubsystems = ""
+    for baseDir in guiSubsystems:
+        if os.path.isdir(baseDir + os.sep + baseDir):
+            guiIncludes += ' -I "../../' + baseDir + os.sep + baseDir + os.sep + '"'
```

Comment:
The use of `../../` and os.sep are somewhat contradictory. Prefer `+= ' -I "' + os.sep.join('..', '..', baseDir, baseDir) + '"'`.

I know this is likely not built on systems that cannot handle '/' as os.sep, but still, os.sep is good practice. :-)

Reply:
Well also we were iterating in GUI subsystems and checking for ROS ones :yum: